packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -120,6 +122,17 @@ class MediaQueryData {
   ///   formatting.
   final bool alwaysUse24HourFormat;
 
+  /// Whether the user is using assitive technologies to interact with the
+  /// application.
+  ///
+  /// This includes screen readers such as TalkBack on Android and VoiceOVer
+  /// on iOS, as well as hardware switches, and more.
+  ///
+  /// See also:
+  ///
+  ///  * [Window.assistiveTechnologyEnabled], for the flag on the window.
```

Comment:
"for the flag on the window" is self-evident from the name. Maybe "which this attribute is based on", or something.